gettingstarted/index.md

```diff
@@ -5,6 +5,67 @@ layout: page
 Getting Started
 ---------------
 
+### Command line generator
+
+The easiest way to begin using courier is via the standalone command line tool. The standalone tool
+generates type safe bindings for any courier-supported language. Assuming Java (JRE) is already
+installed on your system, you can download the binary from
+[Bintray](https://dl.bintray.com/coursera/generic/courier) and install it by doing the following:
+
+~~~sh
+cd /tmp && \
+wget https://dl.bintray.com/coursera/generic/courier && \
+chmod a+x courier && \
+sudo mv courier /usr/local/bin
+~~~
+
+To generate typescript bindings, set up the following project structure:
+
+##### myProject/schemas/org/example/fortune/Fortune.courier
+
+~~~
+namespace org.example.fortune
+
+record Fortune {
+  message: string
+}
+~~~
+
+Then, in the the root of the project (`myProject`), to generate typescript bindings, execute:
+
+~~~
+$ courier ts output /tmp src
```

Comment:
Given the project structure `myProject/schemas/org/example/fortune/Fortune.courier`, this line should probably be `courier ts output /tmp schemas`